[1.21.1] Crash when using iron's spells n spellbooks

It fails when trying to parse compat module for Iron's spells n spellbooks:

```
[13:05:00] [Render thread/ERROR] [minecraft/RegistryDataLoader]: Registry loading errors:
> Errors in registry cold_sweat:item/insulator:
>> Errors in element cold_sweat:irons_spells/fireward_ring:
java.lang.IllegalStateException: Failed to parse cold_sweat:cold_sweat/item/insulator/irons_spells/fireward_ring.json from pack mod/cold_sweat
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.resources.RegistryDataLoader.loadContentsFromManager(RegistryDataLoader.java:243)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.resources.RegistryDataLoader$Loader.loadFromResources(RegistryDataLoader.java:295)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.resources.RegistryDataLoader.lambda$load$2(RegistryDataLoader.java:118)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.resources.RegistryDataLoader.lambda$load$5(RegistryDataLoader.java:138)
	at java.base/java.lang.Iterable.forEach(Iterable.java:75)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.resources.RegistryDataLoader.load(RegistryDataLoader.java:138)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.resources.RegistryDataLoader.mixinextras$bridge$load$43(RegistryDataLoader.java)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.resources.RegistryDataLoader.wrapOperation$dbi000$fabric_resource_conditions_api_v1$captureRegistries(RegistryDataLoader.java:1055)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.resources.RegistryDataLoader.mixinextras$bridge$wrapOperation$dbi000$fabric_resource_conditions_api_v1$captureRegistries$44(RegistryDataLoader.java)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.resources.RegistryDataLoader.wrapOperation$dfb000$fabric_registry_sync_v0$wrapIsServerCall(RegistryDataLoader.java:1552)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.resources.RegistryDataLoader.load(RegistryDataLoader.java:118)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.server.WorldLoader.loadLayer(WorldLoader.java:76)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.server.WorldLoader.loadAndReplaceLayer(WorldLoader.java:82)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.server.WorldLoader.load(WorldLoader.java:35)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.client.gui.screens.worldselection.CreateWorldScreen.openFresh(CreateWorldScreen.java:123)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.client.gui.screens.worldselection.WorldSelectionList.loadLevels(WorldSelectionList.java:192)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.client.gui.screens.worldselection.WorldSelectionList.<init>(WorldSelectionList.java:111)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.client.gui.screens.worldselection.SelectWorldScreen.init(SelectWorldScreen.java:51)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.client.gui.screens.Screen.init(Screen.java:317)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.client.Minecraft.setScreen(Minecraft.java:1057)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.client.gui.screens.TitleScreen.lambda$createNormalMenuOptions$7(TitleScreen.java:161)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.client.gui.components.Button.onPress(Button.java:41)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.client.gui.components.AbstractButton.onClick(AbstractButton.java:47)
	at TRANSFORMER/neoforge@21.1.220/net.neoforged.neoforge.client.extensions.IAbstractWidgetExtension.onClick(IAbstractWidgetExtension.java:36)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.client.gui.components.AbstractWidget.mouseClicked(AbstractWidget.java:144)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.client.gui.components.events.ContainerEventHandler.mouseClicked(ContainerEventHandler.java:38)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.client.gui.screens.TitleScreen.mouseClicked(TitleScreen.java:340)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.client.MouseHandler.lambda$onPress$0(MouseHandler.java:98)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.client.gui.screens.Screen.wrapScreenError(Screen.java:451)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.client.MouseHandler.onPress(MouseHandler.java:95)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.client.MouseHandler.lambda$setup$4(MouseHandler.java:202)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.util.thread.BlockableEventLoop.execute(BlockableEventLoop.java:98)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.client.MouseHandler.lambda$setup$5(MouseHandler.java:202)
	at MC-BOOTSTRAP//me.decce.ixeris.core.glfw.callback_dispatcher.MouseButtonCallbackDispatcher.lambda$onCallback$0(MouseButtonCallbackDispatcher.java:95)
	at MC-BOOTSTRAP//me.decce.ixeris.core.threading.RenderThreadDispatcher.replayQueue(RenderThreadDispatcher.java:30)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.client.Minecraft.handler$fag000$ixeris$replayQueue(Minecraft.java:4023)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.client.Minecraft.runTick(Minecraft.java:1224)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.client.Minecraft.run(Minecraft.java:807)
	at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: java.lang.IllegalArgumentException: No enum constant net.minecraft.world.entity.ai.attributes.AttributeModifier.Operation.ADDITION
	at java.base/java.lang.Enum.valueOf(Enum.java:293)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.world.entity.ai.attributes.AttributeModifier$Operation.valueOf(AttributeModifier.java:25)
	at TRANSFORMER/cold_sweat@2.4-b06g/com.momosoftworks.coldsweat.data.codec.util.AttributeCodecs.lambda$static$0(AttributeCodecs.java:17)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.datafixers.util.Pair.mapFirst(Pair.java:64)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.Decoder$2.lambda$decode$0(Decoder.java:63)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.DataResult$Success.map(DataResult.java:175)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.Decoder$2.decode(Decoder.java:63)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.Codec$2.decode(Codec.java:75)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.Decoder.parse(Decoder.java:18)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.codecs.FieldDecoder.decode(FieldDecoder.java:29)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.MapCodec$2.decode(MapCodec.java:77)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.codecs.RecordCodecBuilder$Instance$5.decode(RecordCodecBuilder.java:325)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.codecs.RecordCodecBuilder$2.decode(RecordCodecBuilder.java:107)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.MapDecoder.lambda$compressedDecode$1(MapDecoder.java:52)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.DataResult$Success.flatMap(DataResult.java:201)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.MapDecoder.compressedDecode(MapDecoder.java:52)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.MapCodec$MapCodecCodec.decode(MapCodec.java:158)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.codecs.ListCodec$DecoderState.accept(ListCodec.java:74)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.JsonOps.lambda$getList$21(JsonOps.java:283)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.codecs.ListCodec.lambda$decode$2(ListCodec.java:45)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.DataResult$Success.flatMap(DataResult.java:201)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.codecs.ListCodec.decode(ListCodec.java:43)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.Decoder.parse(Decoder.java:18)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.codecs.BaseMapCodec.lambda$decode$3(BaseMapCodec.java:34)
	at java.base/java.util.stream.ReduceOps$1ReducingSink.accept(ReduceOps.java:80)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197)
	at java.base/java.util.Iterator.forEachRemaining(Iterator.java:133)
	at java.base/java.util.Spliterators$IteratorSpliterator.forEachRemaining(Spliterators.java:1939)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)
	at java.base/java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:921)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.reduce(ReferencePipeline.java:667)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.codecs.BaseMapCodec.decode(BaseMapCodec.java:30)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.codecs.UnboundedMapCodec.lambda$decode$0(UnboundedMapCodec.java:22)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.DataResult$Success.flatMap(DataResult.java:201)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.codecs.UnboundedMapCodec.decode(UnboundedMapCodec.java:22)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.Decoder$2.decode(Decoder.java:63)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.Codec$2.decode(Codec.java:75)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.Decoder.parse(Decoder.java:18)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.codecs.OptionalFieldCodec.decode(OptionalFieldCodec.java:34)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.MapDecoder$4.decode(MapDecoder.java:94)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.MapCodec$2.decode(MapCodec.java:77)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.codecs.RecordCodecBuilder$Instance$3.decode(RecordCodecBuilder.java:248)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.codecs.RecordCodecBuilder$Instance$5.decode(RecordCodecBuilder.java:322)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.codecs.RecordCodecBuilder$2.decode(RecordCodecBuilder.java:107)
	at TRANSFORMER/cold_sweat@2.4-b06g/com.momosoftworks.coldsweat.data.codec.impl.ConfigData$1.decode(ConfigData.java:65)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.MapDecoder.lambda$compressedDecode$1(MapDecoder.java:52)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.DataResult$Success.flatMap(DataResult.java:201)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.MapDecoder.compressedDecode(MapDecoder.java:52)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.MapCodec$MapCodecCodec.decode(MapCodec.java:158)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.Codec$2.decode(Codec.java:75)
	at TRANSFORMER/neoforge@21.1.220/net.neoforged.neoforge.common.conditions.ConditionalOps$ConditionalDecoder.lambda$decode$9(ConditionalOps.java:197)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.DataResult$Success.map(DataResult.java:175)
	at TRANSFORMER/neoforge@21.1.220/net.neoforged.neoforge.common.conditions.ConditionalOps$ConditionalDecoder.decode(ConditionalOps.java:193)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.Codec$2.decode(Codec.java:75)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.Decoder$2.decode(Decoder.java:63)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.Codec$2.decode(Codec.java:75)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.Decoder.parse(Decoder.java:18)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.resources.RegistryDataLoader.loadElementFromResource(RegistryDataLoader.java:210)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.resources.RegistryDataLoader.loadContentsFromManager(RegistryDataLoader.java:238)
	... 38 more
>> Errors in element cold_sweat:irons_spells/frostward_ring:
java.lang.IllegalStateException: Failed to parse cold_sweat:cold_sweat/item/insulator/irons_spells/frostward_ring.json from pack mod/cold_sweat
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.resources.RegistryDataLoader.loadContentsFromManager(RegistryDataLoader.java:243)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.resources.RegistryDataLoader$Loader.loadFromResources(RegistryDataLoader.java:295)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.resources.RegistryDataLoader.lambda$load$2(RegistryDataLoader.java:118)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.resources.RegistryDataLoader.lambda$load$5(RegistryDataLoader.java:138)
	at java.base/java.lang.Iterable.forEach(Iterable.java:75)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.resources.RegistryDataLoader.load(RegistryDataLoader.java:138)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.resources.RegistryDataLoader.mixinextras$bridge$load$43(RegistryDataLoader.java)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.resources.RegistryDataLoader.wrapOperation$dbi000$fabric_resource_conditions_api_v1$captureRegistries(RegistryDataLoader.java:1055)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.resources.RegistryDataLoader.mixinextras$bridge$wrapOperation$dbi000$fabric_resource_conditions_api_v1$captureRegistries$44(RegistryDataLoader.java)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.resources.RegistryDataLoader.wrapOperation$dfb000$fabric_registry_sync_v0$wrapIsServerCall(RegistryDataLoader.java:1552)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.resources.RegistryDataLoader.load(RegistryDataLoader.java:118)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.server.WorldLoader.loadLayer(WorldLoader.java:76)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.server.WorldLoader.loadAndReplaceLayer(WorldLoader.java:82)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.server.WorldLoader.load(WorldLoader.java:35)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.client.gui.screens.worldselection.CreateWorldScreen.openFresh(CreateWorldScreen.java:123)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.client.gui.screens.worldselection.WorldSelectionList.loadLevels(WorldSelectionList.java:192)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.client.gui.screens.worldselection.WorldSelectionList.<init>(WorldSelectionList.java:111)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.client.gui.screens.worldselection.SelectWorldScreen.init(SelectWorldScreen.java:51)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.client.gui.screens.Screen.init(Screen.java:317)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.client.Minecraft.setScreen(Minecraft.java:1057)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.client.gui.screens.TitleScreen.lambda$createNormalMenuOptions$7(TitleScreen.java:161)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.client.gui.components.Button.onPress(Button.java:41)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.client.gui.components.AbstractButton.onClick(AbstractButton.java:47)
	at TRANSFORMER/neoforge@21.1.220/net.neoforged.neoforge.client.extensions.IAbstractWidgetExtension.onClick(IAbstractWidgetExtension.java:36)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.client.gui.components.AbstractWidget.mouseClicked(AbstractWidget.java:144)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.client.gui.components.events.ContainerEventHandler.mouseClicked(ContainerEventHandler.java:38)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.client.gui.screens.TitleScreen.mouseClicked(TitleScreen.java:340)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.client.MouseHandler.lambda$onPress$0(MouseHandler.java:98)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.client.gui.screens.Screen.wrapScreenError(Screen.java:451)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.client.MouseHandler.onPress(MouseHandler.java:95)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.client.MouseHandler.lambda$setup$4(MouseHandler.java:202)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.util.thread.BlockableEventLoop.execute(BlockableEventLoop.java:98)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.client.MouseHandler.lambda$setup$5(MouseHandler.java:202)
	at MC-BOOTSTRAP//me.decce.ixeris.core.glfw.callback_dispatcher.MouseButtonCallbackDispatcher.lambda$onCallback$0(MouseButtonCallbackDispatcher.java:95)
	at MC-BOOTSTRAP//me.decce.ixeris.core.threading.RenderThreadDispatcher.replayQueue(RenderThreadDispatcher.java:30)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.client.Minecraft.handler$fag000$ixeris$replayQueue(Minecraft.java:4023)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.client.Minecraft.runTick(Minecraft.java:1224)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.client.Minecraft.run(Minecraft.java:807)
	at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: java.lang.IllegalArgumentException: No enum constant net.minecraft.world.entity.ai.attributes.AttributeModifier.Operation.ADDITION
	at java.base/java.lang.Enum.valueOf(Enum.java:293)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.world.entity.ai.attributes.AttributeModifier$Operation.valueOf(AttributeModifier.java:25)
	at TRANSFORMER/cold_sweat@2.4-b06g/com.momosoftworks.coldsweat.data.codec.util.AttributeCodecs.lambda$static$0(AttributeCodecs.java:17)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.datafixers.util.Pair.mapFirst(Pair.java:64)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.Decoder$2.lambda$decode$0(Decoder.java:63)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.DataResult$Success.map(DataResult.java:175)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.Decoder$2.decode(Decoder.java:63)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.Codec$2.decode(Codec.java:75)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.Decoder.parse(Decoder.java:18)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.codecs.FieldDecoder.decode(FieldDecoder.java:29)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.MapCodec$2.decode(MapCodec.java:77)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.codecs.RecordCodecBuilder$Instance$5.decode(RecordCodecBuilder.java:325)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.codecs.RecordCodecBuilder$2.decode(RecordCodecBuilder.java:107)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.MapDecoder.lambda$compressedDecode$1(MapDecoder.java:52)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.DataResult$Success.flatMap(DataResult.java:201)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.MapDecoder.compressedDecode(MapDecoder.java:52)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.MapCodec$MapCodecCodec.decode(MapCodec.java:158)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.codecs.ListCodec$DecoderState.accept(ListCodec.java:74)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.JsonOps.lambda$getList$21(JsonOps.java:283)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.codecs.ListCodec.lambda$decode$2(ListCodec.java:45)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.DataResult$Success.flatMap(DataResult.java:201)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.codecs.ListCodec.decode(ListCodec.java:43)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.Decoder.parse(Decoder.java:18)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.codecs.BaseMapCodec.lambda$decode$3(BaseMapCodec.java:34)
	at java.base/java.util.stream.ReduceOps$1ReducingSink.accept(ReduceOps.java:80)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197)
	at java.base/java.util.Iterator.forEachRemaining(Iterator.java:133)
	at java.base/java.util.Spliterators$IteratorSpliterator.forEachRemaining(Spliterators.java:1939)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)
	at java.base/java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:921)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.reduce(ReferencePipeline.java:667)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.codecs.BaseMapCodec.decode(BaseMapCodec.java:30)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.codecs.UnboundedMapCodec.lambda$decode$0(UnboundedMapCodec.java:22)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.DataResult$Success.flatMap(DataResult.java:201)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.codecs.UnboundedMapCodec.decode(UnboundedMapCodec.java:22)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.Decoder$2.decode(Decoder.java:63)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.Codec$2.decode(Codec.java:75)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.Decoder.parse(Decoder.java:18)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.codecs.OptionalFieldCodec.decode(OptionalFieldCodec.java:34)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.MapDecoder$4.decode(MapDecoder.java:94)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.MapCodec$2.decode(MapCodec.java:77)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.codecs.RecordCodecBuilder$Instance$3.decode(RecordCodecBuilder.java:248)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.codecs.RecordCodecBuilder$Instance$5.decode(RecordCodecBuilder.java:322)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.codecs.RecordCodecBuilder$2.decode(RecordCodecBuilder.java:107)
	at TRANSFORMER/cold_sweat@2.4-b06g/com.momosoftworks.coldsweat.data.codec.impl.ConfigData$1.decode(ConfigData.java:65)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.MapDecoder.lambda$compressedDecode$1(MapDecoder.java:52)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.DataResult$Success.flatMap(DataResult.java:201)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.MapDecoder.compressedDecode(MapDecoder.java:52)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.MapCodec$MapCodecCodec.decode(MapCodec.java:158)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.Codec$2.decode(Codec.java:75)
	at TRANSFORMER/neoforge@21.1.220/net.neoforged.neoforge.common.conditions.ConditionalOps$ConditionalDecoder.lambda$decode$9(ConditionalOps.java:197)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.DataResult$Success.map(DataResult.java:175)
	at TRANSFORMER/neoforge@21.1.220/net.neoforged.neoforge.common.conditions.ConditionalOps$ConditionalDecoder.decode(ConditionalOps.java:193)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.Codec$2.decode(Codec.java:75)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.Decoder$2.decode(Decoder.java:63)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.Codec$2.decode(Codec.java:75)
	at MC-BOOTSTRAP/datafixerupper@8.0.16/com.mojang.serialization.Decoder.parse(Decoder.java:18)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.resources.RegistryDataLoader.loadElementFromResource(RegistryDataLoader.java:210)
	at TRANSFORMER/minecraft@1.21.1/net.minecraft.resources.RegistryDataLoader.loadContentsFromManager(RegistryDataLoader.java:238)
	... 38 more
```

--- a/src/main/resources/data/cold_sweat/cold_sweat/item/insulator/irons_spells/fireward_ring.json
+++ b/src/main/resources/data/cold_sweat/cold_sweat/item/insulator/irons_spells/fireward_ring.json
@@ -9,7 +9,7 @@
       {
         "name": "cs:irons_fireward_protection",
         "amount": 0.45,
-        "operation": "addition"
+        "operation": "add_value"
       }
     ]
   }

--- a/src/main/resources/data/cold_sweat/cold_sweat/item/insulator/irons_spells/frostward_ring.json
+++ b/src/main/resources/data/cold_sweat/cold_sweat/item/insulator/irons_spells/frostward_ring.json
@@ -9,7 +9,7 @@
       {
         "name": "cs:irons_frostward_protection",
         "amount": -0.45,
-        "operation": "addition"
+        "operation": "add_value"
       }
     ]
   }

